### ISSUE TITLE
User problems with repeat dates in Whats On

### ISSUE THREAD

**Issue by GilesGibson** [folklabs/vassallandcoldharbour#36]
New site. Logged on as Giles Gibson.

First event that has been posted has fallen foul of the repeat event issue yet again. 

http://www.vassallandcoldharbour.org.uk/events/father-nature-natural-thinkers-lambeth-country-show

The event lasts for over 24 hours non-stop. I think the wording needs to change and/or the layout drastically re-done to try and reduce this support issue.

**Comment by pmackay** [comment]
It is the Country Show which is all weekend right? :) 

I will try transferring the date text from HH to here - have you noticed any change in this issue on HH since we updated text?

**Comment by GilesGibson** [comment]
yes, it is on for two days, not all weekend. It should have had an entry for Saturday and then repeat for Sunday. 

I think we got another error in the last week on the HH site. There are a few now that make the events look rubbish when viewed in calendar mode.

Not sure what to do really. If we cannot get inside the module to break it up a bit and provide a step by step approach then we are limited.

**Comment by pmackay** [comment]
Just been digging into this. I hadnt actually applied it to HH yet. We'd
had an email conversation in June and I applied the text on Vassall (as the
site we're actively working on). However I recently updated Panopoly and it
removed it patch related to this, so I've reapplied that. Vassall now shows
the text on the date field:

http://www.vassallandcoldharbour.org.uk/node/add/ol-event

Thoughts? Can you test on Tori as discussed? Also maybe email the person
who posted the event above and ask them to check the text?

**Comment by GilesGibson** [comment]
OK, that explains a few things, thanks.
I will raise another issue about being able to contact authors easily.

**Comment by pmackay** [comment]
I've done a quick override on HH with same text. See if it helps...

**Comment by pmackay** [comment]
https://github.com/folklabs/hernehill.org.uk/issues/570 also covers this (just to link up). Need to see if new text helps. Wonder if we could do specific user testing with anyone?

**Comment by djwesto** [comment]
Agree with Paul. I think this is a user testing thing. 

And I also recommend we sit down with Giles too and actually review the entire layout. 

@pmackay, if I was to come up with an alternative layout for the date module, would it be one tpl.php file we'd need to edit?

**Comment by pmackay** [comment]
Customising the date field layout would be more a template preprocess
method or hook_alter, something like that. Not a .tpl file.
